pkg/openshift-sdn/proxy.go

```diff
@@ -43,7 +45,16 @@ func readProxyConfig(filename string) (*kubeproxyconfig.KubeProxyConfiguration,
 	if err := o.Complete(); err != nil {
 		return nil, err
 	}
-	return o.GetConfig(), nil
+
+	config := o.GetConfig()
```

Comment:
Is there a reason we can't disable the endpointslice featuregates in the config file itself, so we don't need the changes you added to `readProxyConfig`?

Otherwise LGTM.

Reply:
"because CI is on fire and I didn't want to try to get a CNO PR in too". But I'll fix that later...